### PR TITLE
Access Rights Documentation

### PR DESCRIPTION
**Pull request by Annamn61**
Access Rights for Streams, Types, StreamViews, Quantities, and UOMs included. 
**This is not available to the public yet, so please do NOT merge immediately**

### CODE REVIEW

File: Documentation/SequentialDataStore/Units_of_Measure.md [hunk 1]
```diff
@@ -721,7 +721,48 @@ The response includes a status code.
 ```csharp
    Task UpdateQuantityOwnerAsync(string quantityId, Trustee quantityOwner);
 ```
-****
+***
+
+## `Get Quantity Access Rights`
+
+Gets the Access Rights associated with the specified quantity for the requesting identity.  For
```
Comment (osimarkrogers) [inline Documentation/SequentialDataStore/Units_of_Measure.md]: Shouldn't "quantity" be referred to as "quantity units of measure" in the intro?

File: Documentation/SequentialDataStore/SDS_Types.md [hunk 2]
```diff
@@ -1614,3 +1614,45 @@ The response includes a status code.
 ```csharp
    Task UpdateTypeOwnerAsync(string typeId, Trustee typeOwner);
 ```
+***
+
+## `Get Type Access Rights`
```
Comment (osimarkrogers) [inline Documentation/SequentialDataStore/SDS_Types.md]: I can't figure out why this didn't turn to bold & blue like the other section headings.  Usually that's from a missing character in a preceding paragraph, but I don't see anything.  Take a look and if you don't find anything, we'll deal with it if it actually shows up in DocFX.

Comment (Annamn61) [reply Documentation/SequentialDataStore/SDS_Types.md]: @osimarkrogers I'm with you, not exactly sure why this is happening. When I view the rich diff though, it seems like all of the correct formatting is in place

Comment (mckenzielee) [reply Documentation/SequentialDataStore/SDS_Types.md]: @Annamn61 please do the docFX build locally to verify that this is formatted correctly.

Comment (Annamn61) [reply Documentation/SequentialDataStore/SDS_Types.md]: @mckenzielee the docfx build does show the correct formatting

**Pull request merged**